spec/features/03_creator_can_update_bourbon_info_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+feature "The bourbon edit page: " do
+  let!(:bourbon) { FactoryGirl.create(:bourbon) }
+  scenario "creator can sees the edit page" do
+    visit bourbon_path(bourbon)
+    click_link "Update"
+    expect(page).to have_content "Update Bourbon Form"
+  end
+
+  scenario "creator edits the bourbon info" do
```

Comment:
You should have a test or tests here for failure cases and not just a successful edit state.